### PR TITLE
chore(package): update deps

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,9 +34,9 @@
     "eslint-plugin-import": "^2.20.1",
     "eslint-plugin-jasmine": "^4.1.0",
     "eslint-plugin-prettier": "^3.1.2",
-    "handlebars": "^4.7.2",
+    "handlebars": "^4.7.3",
```
Comment (saschanaz) [inline package.json]: I think we are keeping this for forks, but are there any active forks remaining?

Comment (marcoscaceres) [reply package.json]: Geonovum is still active. But I've not coordinated with them for a while. I'll see if we can get them to switch over to template strings.

Comment (saschanaz) [reply package.json]: Haven't they migrated to the profile yet? 👀

Comment (marcoscaceres) [reply package.json]: They have, but I think they are still running their fork as I don't think they finished migrating to the profile. They were sending one piece at a time. I'll need to speak to them about restarting that work.

**Pull request merged**